elections/steering/2023/candidate-pacoxu.md

```diff
@@ -0,0 +1,47 @@
+-------------------------------------------------------------
+name: Paco Xu 徐俊杰
+ID: pacoxu
+info:
+
+- employer: DaoCloud
+- slack: pacoxu
+
+-------------------------------------------------------------
+
+## SIGS
+
+- SIG-ClusterLifecyle: kubeadm maintainer
+- SIG-Node: Reviewer
+- SIG-CLI and SIG-Testing
+- WG-LTS
+
+## What I have done
+
+I have worked in the infrastructure team of DaoCloud Enterprise Platform(the Kubernetes-based platform) since 2016. In Kubernetes community, I started my journey as an active kubernetes contributor, mainly working on SIG-Node, and SIG-CLI/SIG-Testing in 2018. I really appreciate the work in the Community and learned a lot from all of you, especially some maintainers in the community. With the help of current maintainers, I'm now one of kubeadm maintainers and SIG Node reviewers.
+In SIG Node, my contributions span various aspects of node management. I have fixed issues related to node resources, and CI failures, and have also upgraded and improved functionalities to enhance node stability and performance.
+
+I've been one of the [most active contributors](https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%203%20years&var-metric=contributions&var-repogroup_name=Kubernetes&var-repo_name=kubernetes%2Fkubernetes&var-country_name=All) to the project over the years.
+
+I won the [2021 Kubernetes Contributor Award](https://www.kubernetes.dev/community/awards/2021/) for both SIG-ClusterLifecyle and SIG-CLI. And I organized KCD Chengdu 2022, and am a speaker in Kubecon EU 2023 and Kubecon China 2021. I am a member of the Program Committee for KubeCon China 2023. My speech in KCD Shanghai 2022 is about an introduction about SIG-Node & kubeadm, and how to participate.
```

Comment:
ClusterLifecyle should be ClusterLifecycle

Reply:
Updated.